src/twisted/conch/test/test_forwarding.py

```diff
@@ -67,15 +42,11 @@ def test_channelOpenHostnameRequests(self):
         sut = forwarding.SSHConnectForwardingChannel(
             hostport=('fwd.example.org', 1234))
         # Patch channel and resolver to not touch the network.
-        sut._reactor = MemoryReactorClock()
-        self.patchHostnameEndpointResolver(
-            request=('fwd.example.org', 1234),
-            response=(AF_INET6 ,'::1'),
-            )
-
+        memoryReactor = MemoryReactorClock()
+        sut._reactor = deterministicResolvingReactor(memoryReactor, ['::1'])
```

Comment:
To avoid losing track of this in the meanwhile, I've [recorded it in a ticket](https://twistedmatrix.com/trac/ticket/8939)